Engine/Pyside_Engine_Python.h

```diff
@@ -17,6 +17,9 @@
  * along with Natron.  If not, see <http://www.gnu.org/licenses/gpl-2.0.html>
  * ***** END LICENSE BLOCK ***** */
 
+//Defined to avoid including some headers when running shiboken which may crash shiboken (particularly boost headers)
```

Comment:
why move this here? this should be inside the include guard

Reply:
I don't know exactly yet but Shiboken (2 and 6) does something strange that it makes _"two passes"_ over this file, and at the second _"pass"_ it ignores the `SBK_RUN` definition. Will see if something can be done instead.